ios/src/TGGeoPolygon.mm

```diff
@@ -0,0 +1,72 @@
+//
+//  TGGeoPolygon.mm
+//  tangram
+//
+//  Created by Karim Naaji on 10/27/16.
+//
+//
+
+#import "TGGeoPolygon.h"
+
+#include <vector>
+
+using Ring = std::vector<TGGeoPoint>;
+
+@interface TGGeoPolygon () {
+    std::vector<Ring> polygons;
```

Comment:
Yes, that's a good call, it seems that we can change that without modifying the current interface of this class and we wouldn't have to rearrange the data before giving it to `markerSetPolygon`.